src/modules/comments.js

```diff
@@ -0,0 +1,29 @@
+import { url3 } from './url.js';
+
+export const postComment = async (MealId, name, message) => {
+  const postStatus = await fetch(url3, {
+    method: 'POST',
+    body: JSON.stringify({
+      item_id: MealId,
+      username: name.value,
+      comment: message.value,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  }).then(() => true)
+    .catch(() => false);
+
+  return postStatus;
+  //   if (name.value && message.value) {
+
+//   }
+};
+
+export const getComments = async (mealId) => {
+  const getStatus = await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/rOpkEBLjDARrzi6r3P9W/comments?item_id=${mealId}`, {
+    method: 'GET',
+  });
+  const commentResponse = await getStatus.json();
+  return commentResponse;
+};
```

Comment:
- [ ] Good job applying most of the requirement in comment popup but there is an issue here, As I checked when you comment on a recipe that doesn't have any comments, then that comment also reflect on all the other recipes that dont have any comment. so the problem is with recipes that dont have any genuine comment but still get comment from other recipe. Kindly consider fixing this bug.
here are pictures of the recipes that dont have comments on their own but other recipes comments reflect on them:
![ty](https://user-images.githubusercontent.com/96683843/219865880-ac6fe441-b92a-40bd-b86a-daea03f345ba.jpg)

![tr](https://user-images.githubusercontent.com/96683843/219865886-5d37c303-174b-458e-9505-8571528838ed.jpg)
Here these two recipes dont have their own comment but the comment that I written on `Tuna Nicoise` appears on them.